public/talos/v1.11/security/talos-security-checklist.mdx

```diff
@@ -0,0 +1,160 @@
+
+For more details, see:
+
+- [Certificate Authority Rotation](../security/ca-rotation)
+- [Generating a Secrets Bundle](../getting-started/prodnotes#step-5%3A-generate-secrets-bundle)
+- [Talos Certificate Management](../security/cert-management)
+- [`talosctl rotate ca` Reference](../reference/cli#talosctl-rotate-ca)
+
+## Enable disk encryption
+
+Encrypting disks helps ensure that sensitive data cannot be accessed if the underlying hardware is lost or compromised. In Talos, you can enable several layers of disk protection:
+
+- **TPM-backed disk encryption**: Uses the node’s TPM to automatically unlock encrypted volumes during boot.
+
+- **SecureBoot + TPM for verified boot paths**: Confirms that the kernel and operating system haven’t been tampered with before the system starts.
+
+- **Encrypted user volumes**: Protects data stored on node-local disks used by workloads.
+
+You can configure disk encryption manually or manage it centrally through Omni.
+
+For more details, see [Disk Encryption](../configure-your-talos-cluster/storage-and-disk-management/disk-encryption).
+
+## Keep Talos machines up to date
+
+Talos has a small and predictable upgrade surface, and each release typically includes important changes such as:
+
+- Kernel security patches
+
+- Container runtime updates
+
+- Enhancements to the Talos API
+
+- CVE and security fixes across the operating system
+
+To stay secure and up to date, make sure you upgrade regularly. You can upgrade individual nodes wit
+
+```bash
+talosctl upgrade --nodes <ip>
+```
+
+See [Upgrading Talos Linux](../configure-your-talos-cluster/lifecycle-management/upgrading-talos) for more information.
+
+## Enforce pod security controls
+
+Talos secures the operating system, but Kubernetes is responsible for enforcing how your pods behave at runtime. To strengthen workload security, make sure you:
+
+- Enable Pod Security Admission (PSA) with at least baseline protections enabled.
+
+- Apply network policies to control which pods can communicate with each other.
+
+- Restrict privileged workloads, including host namespaces, privileged mode, and hostPath mounts.
+
+- Use read-only root filesystems whenever possible to reduce the risk of tampering.
+
+See [Pod Security](../../../kubernetes-guides/security/pod-security) for more information.
+
+## Review network and firewall configuration
+
+Talos exposes only the minimal set of API-server–controlled ports, but you can further reduce your attack surface by tightening network boundaries. Consider using:
+
+- The Talos ingress firewall to restrict traffic directly at the OS layer,
+
+- Cloud or platform-level firewalls such as security groups or VPC rules,
+
+See [Ingress Firewall](../networking/ingress-firewall) for more information.
+
+## Use secureBoot + verified boot
+
+When your hardware supports it:
+
+- Enable SecureBoot
+
+- Use TPM-backed key storage
+
+- Enable Talos’ measured boot features (if applicable)
```

Comment:
Measured boot is just SecureBoot+TPM disk encryption. There's not anything else they need to configure